### PR TITLE
fix(Tile): update outline-offset

### LINKED ISSUE #6306: `Radio Tile Group` outline disappear problem
Hi, I'm Zhuoyue from the IBM Watson Studio UI team and am currently working on developing UI for AutoAI.

The `Radio Tile Group`'s **outline disappears** under 70% Zoom or smaller. This seems to be the bug for Chrome alone, in the safari everything is fine.

**Safari**      
image 
**Chrome**
image

**Safari**      
image
**Chrome**
image

A temporary fix we are using is to add the following code inside the `tilesContainer` of the respective `.scss`, but this only partially solves the problem. If we zoom in 67% or less, we need an even smaller `outline-offset` to make sure the outline shows up correctly. But under smaller `outline-offset`, the 110% or more zoom in will looks weird. So I think this component should be updated for this situation. Thanks!

```css
:global(.bx--tile--is-selected){
	outline-offset: -1.5px !important;
}
```

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #6306

- fixes RadioTile outline bug that can be seen [here](https://github.com/carbon-design-system/carbon/issues/6306) &  below:
Screen Shot 2021-03-31 at 2 44 40 PM

#### Testing / Reviewing

- [ ] Checkout: https://deploy-preview-8258--carbon-components-react.netlify.app/?path=/story/components-tile--radio
- [ ] Zoom view to less than 100% - ex: 90%, 80%, etc.
- [ ] Make sure Tile --> Radio & MultiSelect outlines can be seen
  - Select a tile 
  - Then click out of the container to confirm bug is fixed

### CODE REVIEW

**Comment by tw15egan** [comment]
The radio looks great!

Still seeing some cutoff on the multi select variant
Screen Shot 2021-03-31 at 5 13 34 PM

**Comment by andreancardona** [comment]
> The radio looks great!
> 
> Still seeing some cutoff on the multi select variant
> Screen Shot 2021-03-31 at 5 13 34 PM

Ooo I see! Going to confirm a visual change. Converting this to a draft in the mean time

**Comment by andreancardona** [comment]
@laurenmrice would like your opinion on the visual change to the black border to 2px for the following reasons:

- It matches the focus state 2px outline
- and fixes all the bugs/issues

Thank you!

**Comment by laurenmrice** [comment]
@andreancardona The border for the selected state needs to still be 1px. We use 1px for borders like this across our system unless it is a focus state.

**Comment by andreancardona** [comment]
@tw15egan should be ready for review!

**Review by jnm2377 — APPROVED** [review]
LGTM! :)

**Review by laurenmrice — CHANGES_REQUESTED** [review]
- When there are two selected tiles next to eachother, there should not be an overlap of the tile borders. It should still look like 1px border between the tiles. 
Artboard


- When you select a tile, there is a slight jump of the tile text moving out of place. The text should always remain in the same place not matter what state.
Apr-02-2021 16-00-08

**Comment by andreancardona** [comment]
@laurenmrice I'll make sure to look into the jumping text (I think it's an outline v border issue)

But when there are 2 selected tiles - that is the [current styling](https://react.carbondesignsystem.com/?path=/story/components-tile--multi-select)- is that something we still want to update?
Screen Shot 2021-04-02 at 5 40 59 PM

**Comment by laurenmrice** [comment]
@andreancardona Yes, we want to fix that, that is a bug.

File: packages/components/src/components/tile/_tile.scss [hunk 1]
```diff
@@ -197,6 +198,10 @@
     }
   }
 
+  // .#{$prefix}--tile--selectable {
```
Comment (tw15egan) [inline packages/components/src/components/tile/_tile.scss]: Just need to remove this comment and we should be good to go 👍🏻

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅ 🎉

**Review by laurenmrice — APPROVED** [review]
Looks good ! 🙌

**Pull request merged**